python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -248,18 +249,14 @@ def _get_ray_resources_from_group_spec(
     ray_start_params = group_spec["rayStartParams"]
     # This assumes the Ray container is the first.
     # TODO. Clearly warn users to put the Ray container first when using sidecars.
-    k8s_resource_limits = (
-        group_spec["template"]["spec"]["containers"][0]
-        .get("resources", {})
-        .get("limits", {})
-    )
+    k8s_resources = group_spec["template"]["spec"]["containers"][0].get("resources", {})
```

Comment:
We now need both `limits` and `requests` under the `resources` key. So, here we pass down
`group_spec["template"]["spec"]["containers"][0].get("resources", {})`

instead of

`group_spec["template"]["spec"]["containers"][0].get("resources", {}).get("limits", {})`.